src/aktualizr_lite/main.cc

```diff
@@ -0,0 +1,211 @@
+#include <iostream>
+
+#include <openssl/ssl.h>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
```

Comment:
Normally our linter complains about alphabetizing includes within a category, but sometimes it has unexpected preferences that deviate from that. However, I'd expect you'll need to put openssl after boost to make it happy.